src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -143,6 +148,22 @@ public static void setStandaloneConfigIgnoredToRepoConfigs(
         configs.stream().forEach(config -> config.setStandaloneConfigIgnored(isStandaloneConfigIgnored));
     }
 
+    /**
+     * Ensures that the {@code displayName} of each {@code RepoConfiguration} is unique.
+     */
+    public static void makeDisplayNamesUnique(List<RepoConfiguration> configs) {
```

Comment:
The doc comments should be more comprehensive; something like 

`Appends a unique identifier to the display name if more than one {@link RepoConfiguration} share the same display name`.